cmake_overrides.cmake

```diff
@@ -0,0 +1,1250 @@
+	set(CURL_LIBS "ws2_32" ${CURL_LIBS})
+	SET(HAVE_LIBWINMM 1)
+	set(CURL_LIBS "winmm" ${CURL_LIBS})
+	SET(HAVE_WINDOWS_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "windows.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_WINDOWS_H")
+	SET(HAVE_WINSOCK_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "winsock.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_WINSOCK_H")
+	SET(HAVE_WS2TCPIP_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "ws2tcpip.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_WS2TCPIP_H")
+	SET(HAVE_WINSOCK2_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "winsock2.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_WINSOCK2_H")
+	SET(HAVE_WINCRYPT_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "wincrypt.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_WINCRYPT_H")
+	SET(HAVE_STDIO_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "stdio.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_STDIO_H")
+	SET(HAVE_SYS_FILIO_H 0)
+	SET(HAVE_SYS_IOCTL_H 0)
+	SET(HAVE_SYS_RESOURCE_H 0)
+	SET(HAVE_SYS_TYPES_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "sys/types.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_SYS_TYPES_H")
+	SET(HAVE_SYS_UIO_H 0)
+	SET(HAVE_SYS_UN_H 0)
+	SET(HAVE_SYS_XATTR_H 0)
+	SET(HAVE_ARPA_TFTP_H 0)
+	SET(HAVE_ASSERT_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "assert.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_ASSERT_H")
+	SET(HAVE_CRYPTO_H 0)
+	SET(HAVE_ERR_H 0)
+	SET(HAVE_ERRNO_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "errno.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_ERRNO_H")
+	SET(HAVE_IDN2_H 0)
+	SET(HAVE_IFADDRS_H 0)
+	SET(HAVE_KRB_H 0)
+	SET(HAVE_LIBGEN_H 0)
+	SET(HAVE_LOCALE_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "locale.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_LOCALE_H")
+	SET(HAVE_NETINET_TCP_H 0)
+	SET(HAVE_LINUX_TCP_H 0)
+	SET(HAVE_PEM_H 0)
+	SET(HAVE_POLL_H 0)
+	SET(HAVE_RSA_H 0)
+	SET(HAVE_SSL_H 0)
+	SET(HAVE_STDBOOL_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "stdbool.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_STDBOOL_H")
+	SET(HAVE_STDINT_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "stdint.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_STDINT_H")
+	SET(HAVE_STDIO_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "stdio.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_STDIO_H")
+	SET(HAVE_STROPTS_H 0)
+	SET(HAVE_STDDEF_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "stddef.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_STDDEF_H")
+	SET(HAVE_STDINT_H 1)
+	set(CURL_INCLUDES ${CURL_INCLUDES} "stdint.h")
+	set(CURL_TEST_DEFINES "${CURL_TEST_DEFINES} -DHAVE_STDINT_H")
+	SET(HAVE_SYS_UTSNAME_H 0)
+	SET(SIZEOF_SIZE_T 8)
+	SET(HAVE_SIZEOF_SIZE_T TRUE)
+	SET(SIZEOF_SIZE_T_CODE "#define SIZEOF_SIZE_T 8")
+	SET(SIZEOF_SSIZE_T 0)
+	SET(HAVE_SIZEOF_SSIZE_T FALSE)
+	SET(SIZEOF_SSIZE_T_CODE "#define SIZEOF_SSIZE_T 0")
+	SET(SIZEOF_LONG_LONG 8)
+	SET(HAVE_SIZEOF_LONG_LONG TRUE)
+	SET(SIZEOF_LONG_LONG_CODE "#define SIZEOF_LONG_LONG 8")
```

Comment:
must support win32 builds too (also possibly wrt SIZEOF_TIME_T etc.)

Reply:
Given that we don't provide win32 builds, what's the purpose?

Reply:
not completely breaking the ability to perform certain internal test builds

Reply:
Oh, sure.

> also possibly wrt SIZEOF_TIME_T etc

It seems like `SIZEOF_SIZE_T` is the only thing that 32bit vs 64bit differ in.